graphql-dgs-graphiql-autoconfigure/src/main/java/com/netflix/graphql/dgs/graphiql/autoconfiguration/GraphiQLConfigurer.kt

```diff
@@ -16,27 +16,63 @@
 
 package com.netflix.graphql.dgs.graphiql.autoconfiguration
 
+import com.netflix.graphql.dgs.graphiql.autoconfiguration.GraphiQLConfigurer.Constants.PATH_TO_GRAPHIQL_INDEX_HTML
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Configuration
-import org.springframework.web.servlet.config.annotation.EnableWebMvc
+import org.springframework.core.io.Resource
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
 import org.springframework.web.servlet.config.annotation.ViewControllerRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 import org.springframework.web.servlet.resource.PathResourceResolver
+import org.springframework.web.servlet.resource.ResourceTransformer
+import org.springframework.web.servlet.resource.ResourceTransformerChain
+import org.springframework.web.servlet.resource.TransformedResource
+import java.io.BufferedReader
+import java.io.IOException
+import java.nio.charset.StandardCharsets.UTF_8
+import javax.servlet.http.HttpServletRequest
 
 @Configuration
-@EnableWebMvc
```

Comment:
I am really surprised this has not caused issues yet as it disables SpringBoot web mvc config. I had to remove this when trying to figure out why my tests were not passing.